__docs__/wonder-blocks-testing/_overview_utlities.mdx

```diff
@@ -0,0 +1,14 @@
+import {Meta} from "@storybook/blocks";
+
+<Meta
+    title="Packages / Testing / Utilities / Overview"
+    parameters={{
+        chromatic: {
+            disableSnapshot: true,
+        },
+    }}
```

Comment:
**nit:** This is not needed anymore. SB changed the way MDX pages work as part of v7 or v8 (don't remember which exact version). Now MDX pages are treated as docs-only pages, so Chromatic never take them as reference for snapshots.

Reply:
I've removed this from all the MDX files.